### PR TITLE
Renamed`ContainerFile.container` to `ContainerFile.template` and its instances

### LINKED ISSUE #72: e2e tool: transform ContainerFile.control to ContainerFile.template
Let's rename ContainerFile.control to ContainerFile.template and during the start of the program we execute: `cp ContainerFile.template` to `ContainerFile.control`

https://github.com/containers/qm/blob/main/tests/e2e/ContainerFile.control

Also, we need to add ContainerFile.control into the `gitignore`
https://github.com/containers/qm/blob/main/tests/e2e/.gitignore

### PR DESCRIPTION
**Pull request by adityakode**
Changes :  
Renamed`ContainerFile.container` to `ContainerFile.template` and its instances.
Added `ContainerFile.template` to `.gitignore`

Closes #72

### CODE REVIEW

File: tests/e2e/lib/container [hunk 1]
```diff
@@ -72,7 +72,7 @@ setup_node() {
         nodeID="${1}"
 
         # copy control as template to node${number}
-        cp ContainerFile.control ContainerFile.node"${nodeID}"
+        cp  ContainerFile.node"${nodeID}"
```
Comment (dougsland) [inline tests/e2e/lib/container:75]: looks like you need to fix that one.

**Review by dougsland — COMMENTED** [review]
Also, it's missing one rename in lib/container

```
[root@control e2e]# ./run-test-e2e
[ INFO  ] Starting setup
[ INFO  ] ==============================
[ INFO  ] Cleaning any previous e2e files

[ INFO  ] Preparing ASIL environment
[ INFO  ] ==============================
[ INFO  ] Creating container control [ASIL mode]. It might take some time...
Error: stat /root/foo/qm/tests/e2e/ContainerFile.control: no such file or directory
Error: repository name must have at least one component
[ INFO  ] Creating stub systemd service: container-safety in container control. It might take some time...
[ FAILED ] cannot create container safety in target control
Terminated
[root@control e2e]# grep -rni "ContainerFile.control" *
lib/container:76:        if_error_exit "failed to copy ContainerFile control template to node!"
lib/container:189:        "ContainerFile.control" \
```

**Comment by dougsland** [comment]
@adityakode Thanks for your contribution. I made a small comments regarding the changes. Looks like you also improved the markdown file.

Finally, you need to sign your commit with `git commit -s`, here the note from the bot:
```
There is one commit incorrectly signed off. This means that the author of this commit failed to include a Signed-off-by line in the commit message.

To avoid having PRs blocked in the future, always include Signed-off-by: Author Name <authoremail@example.com> in every commit message. You can also do this automatically by using the -s flag (i.e., git commit -s).
```

Please let us know if you need any help.

**Comment by adityakode** [comment]
Hi @dougsland, made a new PR  with all the changes you suggested. #77

**Pull request closed without merging**